src/remote_cluster.rs

```diff
@@ -14,7 +14,7 @@ pub enum RemoteClusterType {
 
 impl From<Vec<String>> for RemoteClusterType {
     fn from(hostnames: Vec<String>) -> Self {
-        if hostnames.len() == 1 && hostnames[0].contains(&CAPELLA_SRV_SUFFIX.to_string()) {
+        if hostnames.len() == 1 && hostnames[0].contains(&CAPELLA_CONNSTR_PREFIX.to_string()) {
```

Comment:
This means that any cluster registered with a single address and `couchbases://` will be registered as Provisioned? That doesn't seem correct?